src/components/books/UnitBook.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../../redux/books/books';
+
+const UnitBook = ({
+  title, author, id,
+}) => {
+  const dispatch = useDispatch();
+
+  const deleteBook = () => dispatch(removeBook(id));
+
+  return (
+    <article className="bookContainer">
+      <div className="bookContent">
+        <small>Fictional</small>
+        <h2>{title}</h2>
+        <p>{author}</p>
+        <div className="callContainer">
+          <button type="submit">Comments </button>
+          <button type="submit" onClick={deleteBook}>
+            Remove
+          </button>
+          <button type="submit">Edit</button>
+        </div>
+      </div>
+      <div className="progressContainer">
+        <div className="circle" />
+        <div className="percentage">
+          <h1>98%</h1>
+          <p>Completed</p>
+        </div>
+      </div>
+      <div className="updateContainer">
+        <p className="currentChapter">CURRENT CHAPTER</p>
+        <p>Chapter 17</p>
+        <button type="submit" className="progressBtn">
+          UPDATE PROGRESS
+        </button>
+      </div>
+    </article>
+  );
+};
```

Comment:
## Your Book
![image](https://user-images.githubusercontent.com/93039359/232180382-60376593-4028-4587-af7e-90cf4dceec00.png)
## Zeplin's Book
![image](https://user-images.githubusercontent.com/93039359/232181076-3b26cacd-9526-4a14-b86c-67e53b5304ab.png)

- [ ] It would be nice if you compare both images and compare colors, widths, heights and fix all the things according to the Zeplin's design to make your project professional and attractive 😊